### ISSUE TITLE
Change pipeline for release progress tracking

### ISSUE THREAD

**Issue by JohanvdWest** [ZenHubIO/support#1410]
The release reports seem like a very powerful feature. Unfortunately we are unable to use it currently as we only move our issues to closed once the whole release PR is done and approved. 

This means there is "no" progress during the development cycle as our issues go into a Review step and wait for all issues to be done before going to master in the PR and closing the issues.

It would be nice to change a release's pipeline that is used to track progress to something other than closed.

**Comment by kendramoroz** [comment]
Agreed, @JohanvdWest - having the option to define done on the release report is on our radar. Bumping this over to the discussion pipeline. We'll report back once we have more to share on this one!
